docs/chapters/03_booster-architecture.md

```diff
@@ -712,7 +712,144 @@ For more information about queries and how to use them, please check the [GraphQ
+| ne     | true/false | Not equal to |
+
+Example:
+
+```graphql
+query {
+  ProductReadModels(filter: { availability: { eq: true } }) {
+    id
+    sku
+    availability
+    price
+  }
+}
+```
+
+##### Number filters
+
+| Filter |  Value  |           Description |
+| :----- | :-----: | --------------------: |
+| eq     |  Float  |              Equal to |
+| ne     |  Float  |          Not equal to |
+| gt     |  Float  |          Greater than |
+| gte    |  Float  | Greater or equal than |
+| lt     |  Float  |            Lower than |
+| lte    |  Float  |   Lower or equal than |
+| in     | [Float] | Exists in given array |
+
+Example:
+
+```graphql
+query {
+  ProductReadModels(filter: { price: { gt: 200 } }) {
+    id
+    sku
+    availability
+    price
+  }
+}
+```
+
+##### String filters
+
+| Filter     |  Value   |                Description |
+| :--------- | :------: | -------------------------: |
+| eq         |  String  |                   Equal to |
+| ne         |  String  |               Not equal to |
+| gt         |  String  |               Greater than |
+| gte        |  String  |      Greater or equal than |
+| lt         |  String  |                 Lower than |
+| lte        |  String  |        Lower or equal than |
+| in         | [String] |      Exists in given array |
+| beginsWith |  String  | Starts with a given substr |
+| contains   |  String  |    Contains a given substr |
+
+Example:
+
+```graphql
+query {
+  ProductReadModels(filter: { sku: { begingsWith: "jewelry" } }) {
+    id
+    sku
+    availability
+    price
+  }
+}
+```
+
+##### Array filters
+
+| Filter   | Value  |             Description |
+| :------- | :----: | ----------------------: |
+| includes | Object | Includes a given object |
+
+Example:
+
+```graphql
+query {
+  CartReadModels(filter: { itemsIds: { includes: "test-item" } }) {
+    id
+    price
+    itemsIds
+  }
+}
+```
+
+_Note: Right now, with complex properties in Arrays, you just can filter them if you know the exact value of an element but is not possible to filter from a property of the element. As a workaround, you can use an array of ids of the complex property and filter for that property as in the example above._
+
+##### Filter combinators
+
+All the filters can be combined to create a more complex search on the same properties of the ReadModel.
+
+| Filter |     Value     |                                      Description |
+| :----- | :-----------: | -----------------------------------------------: |
+| and    |   [Filters]   |       AND - all filters on the list have a match |
+| or     |   [Filters]   | OR - At least one filter of the list has a match |
+| not    | Filter/and/or |            The element does not match the filter |
+
+Example:
+
+```graphql
+query {
+  CartReadModels(filter: { or: [{ id: { contains: "a" } }, { id: { contains: "b" } }] }) {
+    id
+    price
+    itemsIds
+  }
+}
+```
+
+### Getting and filtering read models data at code level
```

Comment:
This comment is for line 886 of this file (Github doesn't let me put a comment there)

Please, change the "Warning" so that it references the method `Booster.entity` instead of `Booster.fetchEntitySnapshot` 🙏